index.js

```diff
@@ -296,11 +297,17 @@ const GetEmail = {
         console.log('get email intent fired');
         if (send) {
             updateAWSConfig();
-            // This promise might be broken.
-            const sendPromise = queueSendEmail();
+            const sendPromise = new AWS.SES({apiVersion: '2010-12-01'}).sendEmail(emailClient.generateParams(userName, data)).promise();
+            sendPromise.then(
+              function(data) {
+                console.log(data.MessageId);
+              }).catch(
+                function(err) {
+                console.error(err, err.stack);
+              });
         }
         return handlerInput.responseBuilder
-            .speak('sending data...')
+            .speak('Data has been sent. Thank you.')
```

Comment:
Suggestion to switch "data" to "email" or "progress report"?

Reply:
"progress report has been sent" sounds good